projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -55,7 +55,7 @@ def fetch_environment(
     # Local is a special environment where it doesn't need to be defined
     # since it will mirror user's execution context directly.
     if environment_name == "local":
-        if credentials.key_secret and credentials.key_id:
+        if credentials.host in ["cloud", "cloud-eu"]:
```

Comment:
I thought we would just check for it being `is not None`? I am not sure how this behaves now